base/docs/helpdb/Base.jl

```diff
@@ -15,6 +15,18 @@ systemerror
 Fill array `A` with the value `x`. If `x` is an object reference, all elements will refer to
 the same object. `fill!(A, Foo())` will return `A` filled with the result of evaluating
 `Foo()` once.
+
+```jldoctest
+julia> A = zeros(2,3)
+2×3 Array{Float64,2}:
+ 0.0  0.0  0.0
+ 0.0  0.0  0.0
+
+julia> fill!(A, 2.)
+2×3 Array{Float64,2}:
+ 2.0  2.0  2.0
+ 2.0  2.0  2.0
+```
```

Comment:
I like showing the output of `zeros` just so people can see that `fill!` actually *does* something, especially since it's an early docstring in `stdlib`.